spec/serialisers/active_model_serialiser_helper_spec.rb

```diff
@@ -22,6 +24,7 @@ class TestItem < TestItemNoTime
   def initialize(options = {})
     @updated_at = options[:updated_at] || Time.now.utc
```

Comment:
maybe change this to `@enabled_time` to keep it consistent with the other examples.